api/graphql/schema.graphql

```diff
@@ -532,6 +544,24 @@ type Role {
   displayName: LocalizedString @rename(attribute: "display_name")
   description: LocalizedString
   isTeamBased: Boolean @rename(attribute: "is_team_based")
+  # teams and roles: built-in
+  users: [UserPublicProfile!] @morphToMany
```

Comment:
I have a few thoughts about the `users` and `roleAssignments` relationships here.
First: do we need both, or should we just rely on roleAssignments?
Second: they need guarding with `@can` or `@canOnParent` directives since the roles themselves are public but their users are not. But I'm not exactly sure what the guard should be. One easy option is to just remove these relationships from the Role schema, since we don't currently have anywhere that needs to get the list of users by Role. Otherwise, maybe we use something like `@can(ability: 'viewAnyPublicProfile')` and `@can(ability: 'viewAnyTeamMembers', model: 'Team') respectively? (I'm trying to think based on our [permissions](https://docs.google.com/spreadsheets/d/1RC7wufxwjLEvaEunHsfZfOzBefg218dvvzkDwKSiQjI/edit#gid=428263758), though we can always add more permissions if necessary.)

Reply:
Removed in [remove extra relationships](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/69500c02baa6a090cdde1213656bb8672c78251c).

Reply:
Hmmm. Yeah, Role.roleAssignments is a tricky one. 🤔

Reply:
:+1: Added in [Add role policy to view any assignment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/3ade4ec6d4cddb8d3436c52457874b539ada2f1e)